logic/overworld.py

```diff
@@ -463,8 +465,8 @@ def __init__(self, options, world_setup, r):
             hookshot_cave.connect(hookshot_cave_chest, AND(FEATHER, PEGASUS_BOOTS)) # boots jump the gap to the chest
             graveyard_cave_left.connect(graveyard_cave_right, HOOKSHOT, one_way=True) # hookshot the block behind the stairs while over the pit
             swamp_chest.connect(swamp, None)  # Clip past the flower
-            self._addEntranceRequirement("d2", POWER_BRACELET) # clip the top wall to walk between the goponga flower and the wall
-            self._addEntranceRequirement("d2", COUNT(SWORD, 2)) # use l2 sword spin to kill goponga flowers
```

Comment:
Specific reason you removed the sword beam kills?

Reply:
Oh. I messed up there. I'll put it back in.